load/TextStockNumberProcessor.py

```diff
@@ -34,7 +34,7 @@ def validateTextStockNumbersFromController(self, stocknumberFileContentsString,
 			(filenames, actualScript) = self.getSampleUnicodeTextFromS3(s3Client, location, fullPath)
 			stockNumberResultList = self.validateUSXStockList(stockNumberList, filenames, actualScript)
 			resultList.extend(stockNumberResultList)
-		#print("TextStocknumberProcessor.validateTextStockNumbersFromController returning resultList[0]: %s " % (resultList[0])) // for filesets, this throws exception
+		print("TextStocknumberProcessor.validateTextStockNumbersFromController returning resultList[0]: %s " % (resultList))
```

Comment:
did you need this? maybe just leave it commented out

Reply:
@bradflood , I have removed, I think it is no longer necessary.